src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1190.mdx

```diff
@@ -0,0 +1,17 @@
+---
+subject: Infrastructure agent
+releaseDate: '2021-06-29'
+version: 1.19.0
+---
+
+## Notes
+A new version of the agent has been released. Follow standard procedures to [update the Infrastructure agent](/docs/infrastructure/install-configure-manage-infrastructure/update-or-uninstall/update-infrastructure-agent).
+New Relic recommends that you upgrade the agent regularly and at a minimum every 3 months.
```

Comment:
I'd rephrase the sentence to: `We recommend you to upgrade the agent every 3 months.`